.github/workflows/1-main-to-staging.yml

```
@@ -64,7 +64,7 @@ jobs:
 
       - name: Add CODEOWNERS
         run: |
-          echo '@uniswap/web-admins' > CODEOWNERS
+          echo '* @uniswap/web-reviewers' > CODEOWNERS
```

This should still be admins, not reviewers, as we want tighter controls on the `releases/*` branches.
```suggestion
          echo '* @uniswap/web-admins' > CODEOWNERS
```